### PR TITLE
Move dataset generators

### PR DESCRIPTION
**Pull request by katxiao**
Move dataset generators to its own module, so that they can be reused for other automated tests.

- Update dataset generators to inherit from a dataset generator of the relevant data type
- Update the dataset imports (in performance tests) to point to new location

Resolve issue #235

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 @task
 def pytest(c):
     c.run(
-        'pytest ./tests/unit ./tests/integration ./tests/performance/tests '
+        'pytest ./tests/unit ./tests/integration ./tests/performance/tests ./tests/datasets/tests '
```
Comment (amontanez24) [inline tasks.py:13]: The tests here are just tests for the generators themselves correct?

File: tests/datasets/__init__.py [hunk 2]
```diff
@@ -1,10 +1,11 @@
 """Dataset Generators to test the RDT Transformers."""
 
-from tests.performance.datasets import boolean, datetime, numerical
-from tests.performance.datasets.base import BaseDatasetGenerator
+from tests.datasets import boolean, categorical, datetime, numerical
```
Comment (amontanez24) [inline tests/datasets/__init__.py:3]: good catch

**Review by amontanez24 — APPROVED** [review]
LGTM!

File: tasks.py (hunk 1 above)
Comment (csala) [reply tasks.py:13]: That's right

**Review by csala — APPROVED** [review]
LGTM!

**Pull request merged**